app/views/clients/new.html.haml

```diff
@@ -2,6 +2,13 @@
   = simple_form_for [client] do |form|
     %fieldset
       %h3.subtitle Register for the service
+
+      %p
+        Before we begin we need to ask for explicit consent to store your data in alignment with our #{link_to("privacy policy", privacy_policy_path, target: '_blank')}. Outside of the tools listed, we will never share your personal information with third parties without your consent.
+
+      .inline_checkbox
+        = form.input :consent_given, as: :boolean, label: 'I give consent to store my data', required: true
+
```

Comment:
I think this might sit better before the submit button - or at least below the error messages